docs/docs/development/docs.md

```diff
@@ -5,19 +5,115 @@
 
 # Documentation
 
-Write documentation if necessary so that developer, administrators and users
-have an easily accessible way of learning about the collaboration manager.
+It's important to have a good and up to date documentation for the
+Collaboration Manager. Documentation is an accessible way to share knowledge
+for developers, administrators and users. It can also be used by support teams
+to resolve issues more efficiently.
 
-## Setup and Serve the Documentation
+## Development Setup
 
 Navigate to the `docs` folder. With the commands `make install` and
 `make serve` you can build the documentation and find it served under
-`http://127.0.0.1:8081/`. Here you can try your newly written documentation or
-search for something yourself.
+`http://127.0.0.1:8081/`. It's a live-preview of the current version and does
+automatically refresh on file-changes.
 
-## Titles
+## Style guide
+
+### Titles
 
 When writing the documentation we are using title case in the
-[`Chicago Manual of Style` style](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style).
+[`Chicago Manual of Style`](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style).
 In addition personal names/ names of packages are written according to their
 documentation.
+
+### Cross-linking between pages
+
+When referencing other pages in the documentation, use relative links to the
+corresponding `.md` file. `MkDocs` will resolve the link.
+
+!!! warning
+
+    Do not use absolute links to other `.md` files in the same documentation,
+    those will break the links when the documentation is built.
+
+### Move documentation files
+
+When moving documentation files, make sure to add a redirect to the new
+location. This can be done by adding a line to the `mkdocs.yml` file:
+
+```yaml
+plugins:
+  - redirects:
+      redirect_maps:
+        'path/to/old/template.md': 'path/to/new/template.md' # Reason for the move
+```
+
+This ensures that we don't break any links to the old location, which are out
+of our control.
+
+### Tree structure
+
+The documentation is split into three main sections, each with its own
+subsections:
+
+<table>
+    <tr>
+        <th>Path</th>
+        <th>Name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td><b>/user</b></td>
+        <td>User documentation</td>
+        <td>
+            For users of the Collaboration Manager. It contains information about how to use the platform and its features.
+            The content shall be written in a way that it is understandable for non-technical users.
```

Comment:
"Shall" shows up a few times and is a little biblical/archaic for tech docs, I'd swap it for "should"